src/network.cpp

```diff
@@ -73,6 +73,17 @@ String generateOwieStatusJson() {
     out.concat("<tr>");
   }
 
+  const int8_t *thermTemps = relay->getTemperaturesCelsius();
```

Comment:
extract this block into "getTempString" and reuse here and below please.

Reply:
Fixed